.ddev/commands/playwright/playwright

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#ddev-generated
+# Remove the line above if you don't want this file to be overwritten when you run
+# ddev get julienloizelet/ddev-playwright
```

Comment:
As the comment says: If we keep them, we can `ddev get julienloizelet/ddev-playwright` again if there's an update.
If we remove them, they are ours to maintain

Reply:
Ah, it has to include the comma. I thought it was commented out because not needed. Nevermind then. 😉

Reply:
We could technically remove the _additional_ comment since only the `#ddev-generated` is required for this to work, but it adds context as to why this is here to begin with